src/stores/security.md

```diff
@@ -2,7 +2,7 @@
 title: Security
 ---
 
-In this section of the guide, you will learn security best practices, as well as how to manage Admin sessions and credentials, to implement CAPTCHA, and to manage website restrictions.
+In this section of the guide, you will learn [security best practices](https://www.adobe.com/content/dam/cc/en/security/pdfs/Adobe-Magento-Commerce-Best-Practices-Guide.pdf){:target="_blank"}, how to manage Admin sessions and credentials, implement CAPTCHA, and manage website restrictions.
```

Comment:
Since it is no longer in the guide, I think we need to reword this

```suggestion
In this section of the guide, you can access [security best practices](https://www.adobe.com/content/dam/cc/en/security/pdfs/Adobe-Magento-Commerce-Best-Practices-Guide.pdf){:target="_blank"}. You can also learn how to manage Admin sessions and credentials, implement CAPTCHA, and manage website restrictions.
```

Reply:
Fixed.